Assets/MixedRealityToolkit.Providers/WindowsMixedReality/WindowsMixedRealityHandRecorder.cs

```diff
@@ -0,0 +1,83 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Utilities;
+using Microsoft.MixedReality.Toolkit.WindowsMixedReality.Input;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+
+#if WINDOWS_UWP
+using System.IO;
+using System.Threading.Tasks;
+using Windows.Storage;
+using Windows.Storage.Pickers;
+using Windows.Storage.Provider;
+#endif
+
+namespace Microsoft.MixedReality.Toolkit.Input
+{
+
+    /// <summary>
+    /// Record joint positions of a hand and log them for use in simulated hands
+    /// </summary>
+    public class WindowsMixedRealityHandRecorder : MonoBehaviour
+    {
+        private static readonly int jointCount = Enum.GetNames(typeof(TrackedHandJoint)).Length;
+
+        public TrackedHandJoint ReferenceJoint = TrackedHandJoint.IndexTip;
```

Comment:
In keeping with the [coding guidelines](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/CodingGuidelines.html#access-modifiers) to make sure only properties are public.

```suggestion
        public TrackedHandJoint ReferenceJoint { get; set; } = TrackedHandJoint.IndexTip;
```